pkg/deploy/deployment_che.go

```diff
@@ -47,6 +54,33 @@ func NewCheDeployment(cr *orgv1.CheCluster, cheImage string, cheTag string, cmRe
 			},
 		}
 	}
+	if cr.Spec.Server.GitSelfSignedCert {
+        gitSelfSignedCertEnv = corev1.EnvVar{
+            Name: "CHE_GIT_SELF__SIGNED__CERT",
+            ValueFrom: &corev1.EnvVarSource{
+                ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
+                    Key: "ca.crt",
+                    LocalObjectReference: corev1.LocalObjectReference{
+                        Name: "che-git-self-signed-cert",
+                    },
+                    Optional: &optionalEnv,
+                },
+            },
+        }
+        gitSelfSignedCertHostEnv = corev1.EnvVar{
+            Name: "CHE_GIT_SELF__SIGNED__CERT__HOST",
+            ValueFrom: &corev1.EnvVarSource{
+                ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
+                    Key: "githost",
+                    LocalObjectReference: corev1.LocalObjectReference{
+                        Name: "che-git-self-signed-cert",
```

Comment:
It is the same name as above.
Is it correct?

Reply:
yes, it uses the same config-map, but another key